config/setup/templates.go

```diff
@@ -56,6 +56,6 @@ func templatesParse(c *Controller) ([]templates.Rule, error) {
 	return rules, nil
 }
 
-const defaultPath = "/"
+const DefaultTemplatePath = "/"
 
-var defaultExtensions = []string{".html", ".htm", ".tmpl", ".tpl", ".txt"}
+var DefaultTemplateExtensions = []string{".html", ".htm", ".tmpl", ".tpl", ".txt"}
```

Comment:
i exported them so that i could use them in tests . Oh , are they still accessible in the same package without being exported ?